ding/utils/traffic_helper.py

```diff
@@ -0,0 +1,151 @@
+from collections import deque
+from typing import TYPE_CHECKING
+import time
+import os
+import json
+import logging
+from random import random
+import pandas as pd
+import ding.utils
+
+if TYPE_CHECKING:
+    from ding.framework import Parallel
+
+
+class Traffic:
+
+    def __init__(self) -> None:
+        self._file = None
+        self._router = None
+        self._data = None
+
+    def config(
+            self,
+            file_path: str = None,
+            online: bool = False,
+            maxlen: int = 10000,
+            router: "Parallel" = None
+    ) -> "Traffic":
+        """
+        Overview:
+            Enable and change the configuration of a Traffic instance.
+        Arguments:
+            - file_path (:obj:`str`): File path that offline data are to be saved at. \
+                In local mode, it must be set sucessfully for at least once. \
+                In remote mode, no need to set it for worker, but is necessary for master. \
+            - online (:obj:`bool`): To enable online analysis and maintain data in memory.
+            - maxlen (:obj:`int`): Max data size in memory.
+            - router (:obj:`Parallel`): To enable remote mode.
+        """
+
+        if online:
```

Comment:
what's the meaning of this part of code?

Reply:
To ensure parameter that given by user is reasonable.